oak-blob-plugins/src/test/java/org/apache/jackrabbit/oak/plugins/blob/FileCacheTest.java

```diff
@@ -317,15 +317,17 @@ public void evictExplicit() throws Exception {
     public void evictImplicit() throws Exception {
         LOG.info("Started evictImplicit");
 
+        cache = FileCache.build(60 * 1024/* KB */, root, loader, null);
+        
```

Comment:
I think good to add `closer.register(cache);` so, all threads are closed after test.

Reply:
Good finding... Added!